kratos/utilities/normal_calculation_utils.h

```diff
@@ -542,18 +543,32 @@ class KRATOS_API(KRATOS_CORE) NormalCalculationUtils
 
     /**
      * @brief Calculates 3D condition area normals shape sensitivity
-     *
      * @param rCondition    Reference to the targe condition
      */
     static void CalculateNormalShapeDerivative3D(
         ConditionType& rCondition
         );
 
+    /**
+     * @brief This method retrieves the containers
+     * @param  rModelPart The modelpart with the containers to retrieve
+     * @return The corresponding containers
+     * @tparam TContainerType The container type
+     */
     template<class TContainerType>
     TContainerType& GetContainer(ModelPart& rModelPart);
 
+    /**
+     * @brief This method computes the normals considering generic algorithm
+     * @param rModelPart The modelpart with normals to compute
+     * @param ConsiderUnitNormal If consider the unit normal
```

Comment:
```suggestion
     * @param ConsiderUnitNormal In order to consider directly the unit normal instead of the area normal multiplied with a coefficient
```